sdcm/cluster.py

```diff
@@ -4667,8 +4667,9 @@ def kill_stress_thread(self):
     def kill_cassandra_stress_thread(self):
         for loader in self.nodes:
             try:
-                loader.remoter.run(cmd='pgrep -f cassandra-stress | xargs -I{}  kill -TERM -{}',
+                loader.remoter.run(cmd='pgrep -f cassandra.stress | xargs -I{}  kill -TERM {}',
```

Comment:
are you sure about those changes ? i.e. the process is called `cassandra-stress` and the minus there was for killing all the process tree. 

if it's not working we might want to use process group id, like mentioned here: 
https://stackoverflow.com/questions/392022/whats-the-best-way-to-send-a-signal-to-all-members-of-a-process-group

Reply:
cassandra-stress is spawning another process, that has cassandra.stress. Somehow `pgrep -f` returns both processes (I tested it locally and it looks like `.` is a part of regex counted as any sign.). Minus was causing errors like  `kill: (-806457) - No such process`.
Also from recent build it looks that it works.